package.json

```diff
@@ -47,24 +54,28 @@
     "stylint": "1.5.9"
   },
   "dependencies": {
-    "@atlaskit/editor-core": "134.0.1",
+    "@atlaskit/editor-core": "acezard/editor-core-fork#134.0.1",
+    "@atlaskit/icon": "21.1.0",
     "@atlaskit/media-core": "32.1.2",
-    "@atlaskit/media-editor": "38.1.4",
     "@atlaskit/smart-card": "14.6.1",
+    "@babel/preset-env": "^7.14.7",
+    "@babel/preset-typescript": "^7.14.5",
+    "@babel/runtime": "^7.0.0",
     "@material-ui/core": "4",
     "@material-ui/styles": "4.10.0",
+    "classnames": "^2.2.5",
     "cozy-bar": "7.7.8",
     "cozy-client": "23.1.2",
     "cozy-device-helper": "1.8.0",
     "cozy-doctypes": "1.70.0",
     "cozy-flags": "2.1.0",
     "cozy-realtime": "3.13.0",
-    "cozy-scripts": "5.1.1",
+    "cozy-scripts": "acezard/cozy-scripts-ts#master",
```

Comment:
to be changed. You should publish a tag from cozy/cozy-scripts.

Reply:
waiting for PR at the moment